app.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const path = require("path");
+
+const PORT = 3000;
+const app = express();
+
+app.use("/assets", express.static(path.join(__dirname, "assets")));
+
+app.use("/static", express.static(path.join(__dirname, "static")));
+
+app.set("views", path.join(__dirname, "views"));
+
+app.set("view engine", "pug");
+
+app.get("/", (req, res) => {
+  res.render("index", { title: "Home" });
+});
+
+app.listen(PORT, console.log("success"));
```

Comment:
Может быть такое, что порт 3000 занят другим приложением, было бы классно дать возможность задавать порт через команду запуска nodejs скрипта.
```suggestion
app.listen(process.env.PORT || PORT, console.log("success"));
```
Так же "success" может выполнится даже если сервер не создался, так как console.log находится не в колбэке, а во второй параметр у listen по факту передаётся undefined. 

Исправление:
```js
app.listen(process.env.PORT || PORT, () => console.log("success"));
```